ui/src/components/Tiles.js

```diff
@@ -0,0 +1,69 @@
+import React, { PropTypes, Component } from 'react';
+
+export default class Tiles extends Component {
+  render() {
+    var rows = [];
+    var tilesType = this.props.tilesType;
+    this.props.tiles.forEach(function(webtile) {
+      var thumbStyle = {
+        backgroundImage: 'url(' + webtile.imageURI + ')'
+      };
+      var enhancedThumbStyle = {
+        backgroundImage: 'url(' + webtile.enhancedImageURI + ')'
+      };
+      var frecent = [];
```

Comment:
Rather than displaying the lengthy `frecent_sites` of the suggested tiles here, listing a few other fields would be more useful for the traffickers/viewers. Such as,

"adgroup_categories": ["Technology General", "Technology Mobile", ...],
"frequency_caps": {"daily": 1, "total": 5},
"explanation": "A technology content site suggestion"